documentation/3.4/content/release-notes.md

```diff
@@ -57,6 +58,15 @@ draft: false
 
 ### Change log
 
+#### Operator 3.4.6
+
+* Resolved several issues related to WebLogic Deploy Tooling (WDT) models for Model in Image
+  * Support or provide clearer validation errors for cases where `AdminServerName` is set or missing and where there is or is not a matching entry under `Server`.
+  * Improve support for dynamic clusters that do not specify `ServerTemplate` or `DynamicClusterSize`.
+  * Provide clearer validation errors for configured clusters that have no member servers defined under `Server`.
+* Resolved an issue related to an NPE generated while ensuring container port names are not longer than the Kubernetes enforced maximum of 15 characters.
```

Comment:
Minor: The issue with container port names was causing StringIndexOutOfBoundsException instead of NPE.